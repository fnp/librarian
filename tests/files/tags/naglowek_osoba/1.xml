<naglowek_osoba>CZEŚNIK</naglowek_osoba>

<kwestia>
<didaskalia>jakby do siebie</didaskalia>
<strofa>
Piękne dobra w każdym względzie ---/
Lasy --- gleba wyśmienita ---/
Dobrą żoną pewnie będzie ---/
Co za czynsze! --- To kobiéta!.../
Trzy folwarki!
</strofa>...
</kwestia>
