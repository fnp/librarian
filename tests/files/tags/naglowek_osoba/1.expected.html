<div id="book-text">
<h4 id="i1">CZEŚNIK</h4>
<div class="kwestia">
<div class="didaskalia" id="i2">jakby do siebie</div>
<div class="stanza" id="i3">
<a href="#f1" class="wl-num">1</a><div class="wl verse" id="f1">Piękne dobra w każdym względzie —</div>
<div class="wl verse" id="f2">Lasy — gleba wyśmienita —</div>
<div class="wl verse" id="f3">Dobrą żoną pewnie będzie —</div>
<div class="wl verse" id="f4">Co za czynsze! — To kobiéta!...</div>
<a href="#f5" class="wl-num">5</a><div class="wl verse" id="f5">Trzy folwarki!</div>
</div>
</div>
</div>
